api/src/test/java/uk/gov/dhsc/htbhf/claimant/model/UpdatableClaimantFieldTest.java

```diff
@@ -0,0 +1,124 @@
+package uk.gov.dhsc.htbhf.claimant.model;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.dhsc.htbhf.claimant.entity.Address;
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.AddressTestDataFactory.aValidAddress;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.AddressTestDataFactory.aValidAddressBuilder;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.ClaimantTestDataFactory.*;
+
+class UpdatableClaimantFieldTest {
```

Comment:
One thing you could do for bonus credit (obviously optional) is to create a custom assertion class like `ConstraintViolationAssert` and have your assertions read something like: `assertThat(originalClaimant).hasDifferentFirstName();` and `assertThat(originalClaimant).hasNoDifferencesTo(newClaimant)` etc. The tests would read really nicely then, but at current would only help this unit test, so up to you whether you feel like doing it.